### PR TITLE
New HoP Sprites and new HoP clothes

### PR DESCRIPTION
**Pull request by TheCrimsonJupiter**
## About the PR
This PR resprites the papercut-proof gloves, head of personnel's jumpsuit/jumpskirt, and adds new a jumpsuit and jumpskirt option to the Head of Personnel's loadout.

## Why / Balance
I'm not a very big of the current HoP sprites for the jumpsuit, gloves, and hat: so I changed them. 

## Technical details
Changed the sprites for:
head of personnel's jumpsuit
head of personnel's jumpskirt
head of personnel's cap
papercut-proof glove

Added to the HoP's loadout:
head of personnel's mess kit (jumpsuit variant) 
head of personnel's mess kit (jumpskirt variant) 

Added to clothing printer:
head of personnel's mess kit (jumpsuit variant) 
head of personnel's mess kit (jumpskirt variant) 

## Media
image
TWEAKED! My changes make the gloves more discernible, and the makes their legs leg-sized. The shading has also been adjusted everywhere to look more pleasing to the eyes.

image
NEW! The head of personnel's mess kit. A more servicey looking uniform has been given to the HoP. This should make them look at home in the kitchen or other service areas, and also shouldn't get them confused with the Captain.

image
NEW! The new clothing is available in the loadout and can be made at the clothing printer.

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
I'm 80% sure that none of the errors are caused by my sprites. 

**Changelog**

:cl:
- add: New clothes for the HoP

### CODE REVIEW

**Comment by BoskiYourk** [comment]
YOOOO THIS SHIT FUCKS!!!

**Review by DieselMohawk — DISMISSED** [review]

File: Resources/Prototypes/Entities/Clothing/Hands/gloves.yml [hunk 1]
```diff
@@ -116,9 +116,9 @@
   description: Perfect for dealing with paperwork and matters with bureaucracy.
   components:
   - type: Sprite
-    sprite: Clothing/Hands/Gloves/hop.rsi
+    sprite: _Harmony/Clothing/Hands/Gloves/hop.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Hands/gloves.yml:119]:
```suggestion
    sprite: _Harmony/Clothing/Hands/Gloves/hop.rsi # Harmony change. Resprite
```

Upstream file changes require a comment.

File: Resources/Prototypes/Entities/Clothing/Hands/gloves.yml [hunk 2]
```diff
@@ -116,9 +116,9 @@
...
   - type: Clothing
-    sprite: Clothing/Hands/Gloves/hop.rsi
+    sprite: _Harmony/Clothing/Hands/Gloves/hop.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Hands/gloves.yml:121]:
```suggestion
    sprite: _Harmony/Clothing/Hands/Gloves/hop.rsi # Harmony change. Resprite
```

Upstream file changes require a comment.

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml [hunk 3]
```diff
@@ -322,9 +322,9 @@
   description: A grand, stylish head of personnel's cap.
   components:
   - type: Sprite
-    sprite: Clothing/Head/Hats/hopcap.rsi
+    sprite: _Harmony/Clothing/Head/Hats/hopcap.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Head/hats.yml:325]:
```suggestion
    sprite: _Harmony/Clothing/Head/Hats/hopcap.rsi # Harmony change. Resprite
```

Upstream file changes require a comment.

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml [hunk 4]
```diff
@@ -322,9 +322,9 @@
...
   - type: Clothing
-    sprite: Clothing/Head/Hats/hopcap.rsi
+    sprite: _Harmony/Clothing/Head/Hats/hopcap.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Head/hats.yml:327]:
```suggestion
    sprite: _Harmony/Clothing/Head/Hats/hopcap.rsi # Harmony change. Resprite
```

Upstream file changes require a comment.

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 5]
```diff
@@ -170,9 +170,9 @@
   description: Rather bland and inoffensive. Perfect for vanishing off the face of the universe.
   components:
   - type: Sprite
-    sprite: Clothing/Uniforms/Jumpskirt/hop.rsi
+    sprite: _Harmony/Clothing/Uniforms/Jumpskirt/hop.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:173]:
```suggestion
    sprite: _Harmony/Clothing/Uniforms/Jumpskirt/hop.rsi # Harmony change. Resprite
```
Upstream file changes require a comment.

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 6]
```diff
@@ -170,9 +170,9 @@
...
   - type: Clothing
-    sprite: Clothing/Uniforms/Jumpskirt/hop.rsi
+    sprite: _Harmony/Clothing/Uniforms/Jumpskirt/hop.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml:175]:
```suggestion
    sprite: _Harmony/Clothing/Uniforms/Jumpskirt/hop.rsi # Harmony change. Resprite
```
Upstream file changes require a comment.

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 7]
```diff
@@ -325,9 +325,9 @@
   description: Rather bland and inoffensive. Perfect for vanishing off the face of the universe.
   components:
   - type: Sprite
-    sprite: Clothing/Uniforms/Jumpsuit/hop.rsi
+    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:328]:
```suggestion
    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi # Harmony change. Resprite
```
Upstream file changes require a comment.

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 8]
```diff
@@ -325,9 +325,9 @@
...
   - type: Clothing
-    sprite: Clothing/Uniforms/Jumpsuit/hop.rsi
+    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:330]:
```suggestion
    sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi # Harmony change. Resprite
```
Upstream file changes require a comment.

File: Resources/Textures/_Harmony/Clothing/Head/Hats/hopcap.rsi/meta.json [hunk 9]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from the tg station atcommithttps://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e and then resprited by TadJohnson00 and Timemaster99 for DeltaV atcommithttps://github.com/DeltaV-Station/Delta-v/pull/1501, and then resprited a third time by CrimsonJupiter for Harmony",
```
Comment (FluffMe) [inline Resources/Textures/_Harmony/Clothing/Head/Hats/hopcap.rsi/meta.json:4]: * Multiple spaces missing in copyright.
```suggestion
  "copyright": "Taken from the tg station at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e and then resprited by TadJohnson00 and Timemaster99 for DeltaV at commit https://github.com/DeltaV-Station/Delta-v/pull/1501, and then resprited a third time by CrimsonJupiter for Harmony",
```

* `at commit https://github.com/DeltaV-Station/Delta-v/pull/1501` -- PR request is not a commit. Find a commit it was merged to master in.

File: Resources/Textures/_Harmony/Clothing/Hands/Gloves/hop.rsi/meta.json [hunk 10]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Icon and inhand drawn by Ubaser and takem from this atcommithttps://github.com/space-wizards/space-station-14/pull/21915 , equipped sprite made by CrimsonJupiter for use in Harmony",
```
Comment (FluffMe) [inline Resources/Textures/_Harmony/Clothing/Hands/Gloves/hop.rsi/meta.json:4]: * Multiple typos in copyright.
* `taken from this` from what?
* `at commit <pr link>` PR is not a commit.

File: Resources/Textures/_Harmony/Clothing/Uniforms/Jumpskirt/hop.rsi/meta.json [hunk 11]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Originally taken from tgstation atcommithttps://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 icon, inhand, and equipped resprites by CrimsonJupiter for Harmony.",
```
Comment (FluffMe) [inline Resources/Textures/_Harmony/Clothing/Uniforms/Jumpskirt/hop.rsi/meta.json:4]:
```suggestion
  "copyright": "Originally taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 icon, inhand, and equipped resprites by CrimsonJupiter for Harmony.",
```

File: Resources/Textures/_Harmony/Clothing/Uniforms/Jumpskirt/hopmesskit.rsi/meta.json [hunk 12]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by TadJohnson00 and Timemaster99 for DeltaV atcommithttps://github.com/DeltaV-Station/Delta-v/pull/1501, resprited by CrimsonJupiter for Harmony.",
```
Comment (FluffMe) [inline Resources/Textures/_Harmony/Clothing/Uniforms/Jumpskirt/hopmesskit.rsi/meta.json:4]: * Multiple spaces missing in copyright
```suggestion
  "copyright": "Made by TadJohnson00 and Timemaster99 for DeltaV at commit https://github.com/DeltaV-Station/Delta-v/pull/1501, resprited by CrimsonJupiter for Harmony.",
```
* `at commit https://github.com/DeltaV-Station/Delta-v/pull/1501` PR request is not a commit. Find a commit it was merged to master in.

File: Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi/meta.json [hunk 13]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Originally taken from tgstation atcommithttps://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 icon, inhand, and equipped resprites by CrimsonJupiter for Harmony.",
```
Comment (FluffMe) [inline Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/hop.rsi/meta.json:4]: Multiple spaces missing in copyright
```suggestion
  "copyright": "Originally taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039 icon, inhand, and equipped resprites by CrimsonJupiter for Harmony.",
```

File: Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/hopmesskit.rsi/meta.json [hunk 14]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by TadJohnson00 and Timemaster99 for DeltaV atcommithttps://github.com/DeltaV-Station/Delta-v/pull/1501, resprited by CrimsonJupiter for Harmony.",
```
Comment (FluffMe) [inline Resources/Textures/_Harmony/Clothing/Uniforms/Jumpsuit/hopmesskit.rsi/meta.json:4]: * Multiple spaces missing in copyright
```suggestion
  "copyright": "Made by TadJohnson00 and Timemaster99 for DeltaV at commit https://github.com/DeltaV-Station/Delta-v/pull/1501, resprited by CrimsonJupiter for Harmony.",
```
* `at commit https://github.com/DeltaV-Station/Delta-v/pull/1501` PR request is not a commit. Find a commit that merged it into master.

**Review by FluffMe — CHANGES_REQUESTED** [review]

File: Resources/Textures/_Harmony/Clothing/Hands/Gloves/hop.rsi/meta.json (hunk 10 above)
Comment (TheCrimsonJupiter) [reply Resources/Textures/_Harmony/Clothing/Hands/Gloves/hop.rsi/meta.json]: > * Multiple typos in copyright.
> * `taken from this` from what?
> * `at commit <pr link>` PR is not a commit.

Every single mention of a pull has been replaced with a commit in the "at commit <link>" format that you used. I hope this works!

**Review by FluffMe — APPROVED** [review]

**Review by DieselMohawk — APPROVED** [review]

**Pull request merged**